Sources/MintKit/PackageReference.swift

```diff
@@ -23,6 +23,9 @@ public class PackageReference {
             if packageParts[1].contains(":") {
                 repo = [packageParts[0], packageParts[1]].joined(separator: "@")
                 version = ""
+            } else if packageParts[0].contains("ssh://") {
```

Comment:
Actually should this be the following?
```suggestion
            } else if packageParts[0].contains("ssh://git") {
```

Reply:
The part between the SSH scheme and the `@` is the login name to the server.
In my case, it was `git`, but it can be anything.
See [SSH protocol](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols)

Reply:
Right, but ssh can be used without a login name can't it? eg `ssh://server.com/user/project.git`. In that case we would lose the version if it was specified. This function is parsing out the '@' symbols and not ssh specifically. It should probably use some smarter regex.

But what you have here now is better than it was before, as it fixes `ssh://git@s`, so I'll merge 😄👍